frontend/src/lib/Buttons.ts

```diff
@@ -258,7 +283,7 @@ export default class Buttons {
       `be stopped if it's running when it's archived. Use the Restore action to restore the ` +
       `run${s(selectedIds)} to ${selectedIds.length === 1 ? 'its' : 'their'} original location.`,
       useCurrent,
-      id => Apis.runServiceApi.archiveRun(id),
+      Apis.runServiceApi.archiveRun,
```

Comment:
why this change?

Reply:
It was a style changed proposed by me, it can be reverted for readability.

Reply:
I believe there is actually a difference between using the arrow notation as opposed to just passing the function itself. The arrow notation captures the `this` of the surrounding scope whereas if just the function is passed, `this` is limited to the internal scope of the function

Reply:
I hadn't considered that, I believe you are correct. This should be reverted as @rileyjbauer suggested.